test/e2e/common/setup-kubestellar.sh

```diff
@@ -83,6 +91,20 @@ SRC_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
 COMMON_SRCS="${SRC_DIR}/../common"
 source "$COMMON_SRCS/setup-shell.sh"
 
+# Get scenario configuration
+scenario_config=($(get_scenario_config "$TEST_SCENARIO"))
+if [ $? -ne 0 ]; then
+    echo "Invalid scenario: $TEST_SCENARIO" >&2
+    list_scenarios
+    exit 1
+fi
+
+# Parse scenario configuration
+declare -A config
```

Comment:
On my Mac:

```console
bash-3.2$ declare -A config
bash: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
```

Reply:
Mac users are blighted with version 3 of bash, which does not support associative arrays. We should not be in the business of telling users to make a major version change in their system software.